artemis/src/test/java/tech/pegasys/artemis/cli/BeaconNodeCommandTest.java

```diff
@@ -315,6 +363,12 @@ private ArtemisConfiguration getResultingArtemisConfiguration() {
     return configCaptor.getValue();
   }
 
+  private ArtemisConfiguration getArtemisConfigurationFromArguments(List<String> arguments)
```

Comment:
nit: If you used varargs you could avoid a lot of `List.of` calls above and `arguments` would be a `String[]` so not need conversion below.

Reply:
good call